scripts/us_census/pep/Population_Estimate_by_ASR/county_1970_1979.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This Python Script is
+for County Level Data
+1970-1979
+'''
+import os
+import json
+import pandas as pd
+
+
+def county1970():
+    '''
+   This Python Script Loads csv datasets
+   from 1970-1979 on a County Level,
+   cleans it and create a cleaned csv
+   '''
+    # Getting input URL from the JSON file
+    _URLS_JSON_PATH = os.path.dirname(
+        os.path.abspath(__file__)) + os.sep + "county.json"
+    _URLS_JSON = None
+    with open(_URLS_JSON_PATH, encoding="UTF-8") as file:
+        _URLS_JSON = json.load(file)
+    _url = _URLS_JSON["1970-79"]
+    _cols=['Year','geo_ID','Race',0,1,2,3,4,5,6,7\
+                      ,8,9,10,11,12,13,14,15,16,17]
+    final_df = pd.DataFrame()
```

Comment:
Here and everywhere: you use "df, df1, df2, final_df' in many of these files/functions. Can we have better descriptive names? Also, would it be possible to create helper functions to do the different types of DataFrame processing? For instance, a function which does all types of "race" related processing and a separate one for "age" related processing. I know that might not be 100% possible everywhere, but it is not easy to follow/read the code right now and it appears quite fragile.

Reply:
The df names have been changed and more easy to understand now. A comment has also been added along with the definition for better understanding.